bot/utils/blurple_formatter.py

```diff
@@ -0,0 +1,717 @@
+import ast
+import random
+from collections.abc import Generator
+from random import randrange
+
+max_line_length = 50
```

Comment:
Shouldn't these be SCREAMING_CASE and have two lines between them (where you have empty lines at all, that is)?

For the latter case, as long as the linter is happy there I am happy.